config/uberon.yml

```diff
@@ -21,17 +21,11 @@ entries:
 - exact: /phenoscape-ext.owl
   replacement: http://phenoscape.svn.sourceforge.net/svnroot/phenoscape/trunk/vocab/edit/phenoscape-ext.owl
 
-- exact: /composite-vertebrate.owl
-  replacement: https://github.com/obophenotype/uberon/releases/download/v2021-10-01/composite-vertebrate.owl
+- exact: /ext.owl
+  replacement: https://github.com/obophenotype/uberon/releases/latest/download/uberon.owl
```

Comment:
@jamesaoverton ext.owl is redirected to uberon.owl deliberately - they are now one and the same.